app/src/main/res/raw/sessions_en.json

```diff
@@ -23,6 +23,7 @@
     "language_id": "ja",
     "slide_url": "https://speakerdeck.com/player/b2127cb508e94238b114512447997aeb",
     "movie_url": "https://www.youtube.com/watch?v=kArtZzDz1xU",
+    "movie_dash_url": "https://manifest.googlevideo.com/api/manifest/dash/initcwndbps/2632500/id/o-AE9RZgFZZANkR7IkphApmRboitWzzC4-tsg9NhuCOb5K/pl/17/source/youtube/mv/m/key/yt6/signature/96E2E5BF1EF5F2E36FE4BF385FBA675121D8A26D.D6B0F02FC33138432A63AE0B0D1AB39DE8D7986B/mm/31/ip/112.138.237.132/mn/sn-8pjuv35p-ogul/ms/au/sver/3/sparams/as%2Chfr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Cmm%2Cmn%2Cms%2Cmv%2Cpcm2cms%2Cpl%2Cplayback_host%2Crequiressl%2Csource%2Cexpire/itag/0/ipbits/0/as/fmp4_audio_clear%2Cwebm_audio_clear%2Cwebm2_audio_clear%2Cfmp4_sd_hd_clear%2Cwebm2_sd_hd_clear/expire/1468186731/pcm2cms/yes/requiressl/yes/hfr/1/mt/1468164822/upn/dRZOVDIaTsg/playback_host/r3---sn-8pjuv35p-ogul.googlevideo.com/fexp/3300115%2C3300130%2C3300164%2C3313267%2C9406989%2C9416126%2C9416891%2C9422596%2C9428398%2C9431012%2C9433096%2C9433380%2C9433946%2C9435526%2C9435876%2C9437066%2C9437552%2C9439652",
```

Comment:
The following is the trick... :trollface: You can get information from http://www.youtube.com/get_video_info?&video_id=ID and decode it and finally get the DASH url 😅 

http://stackoverflow.com/a/31709557/1288107